### PR TITLE
Allow minimum input tool bar height to be configurable in properties

### PR DESCRIPTION
**Pull request by erenn16**

### CODE REVIEW

File: src/GiftedChat.js [hunk 1]
```diff
@@ -489,6 +489,7 @@ GiftedChat.defaultProps = {
   renderTime: null,
   user: {},
   bottomOffset: 0,
+  minInputToolbarHeight: 0,
```
Comment (dgdavid) [inline src/GiftedChat.js]: Why not put here the value of `MIN_INPUT_TOOLBAR_HEIGHT` and remove it? This also will allow get rid of `setMinInputToolbarHeight` and `this._minInputToolBarHeight`, using directly `this.props.minInputToolbarHeight` into `getMinInputToolbarHeight`.

File: README.md [hunk 2]
```diff
@@ -124,6 +124,7 @@ See [example/App.js](example/App.js)
 - **`renderAccessory`** _(Function)_ - renders a second line of actions below the message composer
 - **`onPressActionButton`** _(Function)_ - callback to perform custom logic when the Action button is pressed (the default `actionSheet` will not be used)
 - **`bottomOffset`** _(Integer)_ - distance of the chat from the bottom of the screen, useful if you display a tab bar
+- **`minInputToolbarHeight`** _(Integer)_ - minimum height of the input toolbar
```
Comment (dgdavid) [inline README.md]: Don't forget to specify the default value.

**Comment by kfiroo** [comment]
Hey guys, sorry I was away for a month.
What's up with this PR? can I merge? @dgdavid @erenn16

**Comment by erenn16** [comment]
@kfiroo I applied the changes that @dgdavid requested.

**Comment by dgdavid** [comment]
@kfiroo @erenn16 LGTM 👍

**Pull request merged**